README.md

```diff
@@ -33,18 +33,21 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
 
 ### Dev Deployment 
 1. Setup your local machine to credentials to deploy to the AWS Account
-2. Deploy the ci-config-stackusing the with one of the following (takes ~1 minute to deploy) - 
-   1. `npm run cdk deploy CI-Config-Dev` or,
-   2. `cdk deploy CI-Config-Dev`
-3. [Optional](#ssl-configuration) Configure the elements of the config stack for SSL configuration
-4. [Optional](#setup-openid-connect-oidc-via-federate) Configure the elements setting up oidc via federate
-5. Deploy the ci-stack, takes ~10 minutes to deploy (parameter values depend on step 3 and step 4)
+1. Deploy the bootstrap stack by running following command that sets up required resources to create the stacks. [More info](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html):
+   1. `npm run cdk bootstrap -- -c useSsl=false -c runWithOidc=false`
+   1. `cdk boostrap -c useSsl=false -c runWithOidc=false` 
```

Comment:
I meant that we only show the top version of this call for consistency 
```suggestion
   1. `npm run cdk bootstrap -- -c useSsl=false -c runWithOidc=false`
```

Reply:
*For all commands

Reply:
Done